frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -64,6 +64,8 @@ class Slider extends React.PureComponent<Props, State> {
 
   private sliderRef = React.createRef<HTMLDivElement>()
 
+  private thumbValueRef = React.createRef<HTMLDivElement>()
```

Comment:
Just want to confirm that it is reasonable and not bad practice to create two refs here.
For context, I need `thumbValueRef` as well so that I can access both the parent element (`Slider`) and one of its children (`StyledThumbValue`) easily in the same function. These are used in `thumbValueAlignment()` to check `StyledThumbValue`'s position relative to `Slider`'s position.

Reply:
All good here